Documentation/doc/Documentation/advanced/Installation.txt

```diff
@@ -137,7 +137,9 @@ Providing information and pressing *Configure* goes on until
 all entries are grayed. You are now ready to press *Generate*. Once this is
 done, you can quit `cmake-gui`.
 
-If you do not need to debug, you should set the variable `CMAKE_BUILD_TYPE` to `Release`.
+Without an explicit `CMAKE_BUILD_TYPE`, `CMake` does not add any additional build flags.
+This will result in a default build without any optimizations and without any debug symbols.
+You should set the variable `CMAKE_BUILD_TYPE` to `Release` or `Debug` to enable optimizations or debug symbols.
```

Comment:
```suggestion
This will result in a default build without any debug symbols, and without any optimizations.
You should set the variable `CMAKE_BUILD_TYPE` to `Debug` to compile with debug symbols, or `Release` to enable optimizations.
```